src/integration-test/java/com/commercetools/project/sync/ProductSyncWithNestedReferencesIT.java

```diff
@@ -8,12 +8,7 @@
 import static com.commercetools.project.sync.util.IntegrationTestUtils.createReferenceObject;
 import static com.commercetools.project.sync.util.SphereClientUtils.CTP_SOURCE_CLIENT_CONFIG;
 import static com.commercetools.project.sync.util.SphereClientUtils.CTP_TARGET_CLIENT_CONFIG;
-import static com.commercetools.project.sync.util.TestUtils.assertCartDiscountSyncerLoggingEvents;
-import static com.commercetools.project.sync.util.TestUtils.assertCategorySyncerLoggingEvents;
-import static com.commercetools.project.sync.util.TestUtils.assertInventoryEntrySyncerLoggingEvents;
-import static com.commercetools.project.sync.util.TestUtils.assertProductSyncerLoggingEvents;
-import static com.commercetools.project.sync.util.TestUtils.assertProductTypeSyncerLoggingEvents;
-import static com.commercetools.project.sync.util.TestUtils.assertTypeSyncerLoggingEvents;
+import static com.commercetools.project.sync.util.TestUtils.*;
```

Comment:
Suggest to use specific import.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/b416d543805c81bd8d34cddd067761e148bba905